app/controllers/feeds_controller.rb

```diff
@@ -13,4 +13,9 @@ def show
   def repository_params
     { owner: params[:owner], repo: params[:repo] }
   end
+
+  def repository
```

Comment:
Since this wraps `@repository` in a presenter, the method should be called `presnted_repository` or similar. The helper method however can stay the same.